importexport/guitarpro/importgtp-gp6.cpp

```diff
@@ -1224,6 +1224,8 @@ Fraction GuitarPro6::readBeats(QString beats, GPPartInfo* partInfo, Measure* mea
                                                 Staff* staff        = note->staff();
                                                 int fretNumber      = fretNum.toInt();
                                                 int musescoreString = staff->part()->instrument()->stringData()->strings() - 1 - stringNum.toInt();
+                                                if (fretNumber > 0 && musescoreString == 4)
+                                                      fretNumber = staff->part()->instrument()->stringData()->adjustBanjo5thFret(fretNumber);
```

Comment:
How does this work on any plucked string instrument with more than 4 strings that is not a Banjo?

Reply:
adjustBanjo5thFret() makes sure number of strings == 5 and uses 'startFret' which is only set for 5-string banjos.